### PR TITLE
- add relaunch capability #275

### PR DESCRIPTION
**Pull request by mgerb**
I have some Angular experience so I decided to take matters into my own hands. Translations still need to be added. Let me know what you think.

All tests pass and I also ran the linter.

image

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -197,6 +197,13 @@ function createTray(): Tray {
         {
             label: 'Exit', type: 'normal',
             click: () => app.quit()
+        },
```
Comment (Kyusung4698) [inline main.ts]: I'd keep the exit button as last entry by moving the relaunch button one up.

File: src/app/layout/component/user-settings-dialog/user-settings-dialog.component.spec.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { UserSettingsFeatureContainerComponent } from '../user-settings-feature-
 import { UserSettingsFormComponent } from '../user-settings-form/user-settings-form.component';
 import { UserSettingsDialogComponent } from './user-settings-dialog.component';
 
-class ElectronProviderFake {
+export class ElectronProviderFake {
```
Comment (Kyusung4698) [inline src/app/layout/component/user-settings-dialog/user-settings-dialog.component.spec.ts]: I'd not export a class inside a spec file. In order to keep the import space safe with just actual classes. For now i'd just copy this class to the next test.

File: src/app/layout/component/user-settings-form/user-settings-form.component.spec.ts [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ElectronProvider } from '@app/provider';
 import { TranslateFakeLoader, TranslateLoader, TranslateModule } from '@ngx-translate/core';
 import { SharedModule } from '@shared/shared.module';
 import { UserSettingsFormComponent } from './user-settings-form.component';
+import { ElectronProviderFake } from '../user-settings-dialog/user-settings-dialog.component.spec';
```
Comment (Kyusung4698) [inline src/app/layout/component/user-settings-form/user-settings-form.component.spec.ts]: Copy `ElectronProviderFake` for now instead of export/ import.

**Comment by mgerb** [comment]
I addressed the few minor issues and force pushed to my branch. Thanks!

**Pull request merged**